gcsfs/core.py

```diff
@@ -835,10 +835,12 @@ async def _glob(self, path, prefix="", **kwargs):
             prefix = path[:ind].split("/")[-1]
         return await super()._glob(path, prefix=prefix, **kwargs)
 
-    async def _ls(self, path, detail=False, prefix="", versions=False, **kwargs):
+    async def _ls(self, path, detail=False, refresh=False, prefix="", versions=False, **kwargs):
```

Comment:
Let's put the new kwarg last, so that if anyone is relying on the order, they don't get broken
```suggestion
    async def _ls(self, path, detail=False, prefix="", versions=False, refresh=False, **kwargs):
```